src/test/java/teammates/test/cases/pagedata/InstructorFeedbackSessionsPageDataTest.java

```diff
@@ -50,10 +52,12 @@ public void testInitWithoutDefaultFormValues() {
         List<CourseAttributes> courses = getCoursesForInstructor(instructorsForUser);
 
         List<FeedbackSessionAttributes> fsList = getFeedbackSessionsListForInstructor(instructorsForUser);
-        List<FeedbackSessionAttributes> recoveryFsList = new ArrayList<>();
+        List<FeedbackSessionAttributes> recoveryFsList = getRecoveryFeedbackSessionsListForInstructor(instructorsForUser);
```

Comment:
Didn't we refine the terminology from `Recovery` to something else, or is it to be done in the future?